tsconfig.json

```diff
@@ -15,6 +15,11 @@
     "lib": [
       "es2018",
       "dom"
-    ]
+    ],
+    "baseUrl": "src",
+    "paths": {
+      "@app-services/*": ["app/core/services/*"],
+      "@app-environments/*": ["app/environments/*"],
```

Comment:
you can add just @app =)

Reply:
i know, but such naming is more understandable for me)